soc/nxp/imxrt/imxrt118x/Kconfig

```diff
@@ -2,6 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_IMXRT118X
+	select CPU_HAS_CUSTOM_FIXED_SOC_MPU_REGIONS if SOC_MIMXRT1189_CM7
```

Comment:
Where do we pull in a custom MPU region definition for the RT1180 CM7 core?

Reply:
Sorry, I forget to delete the kconfig configuration. Originally I use custom mpu region definition way to support MPU configuration. Decsny tell me there is no need to use custom mpu way, so code have been updated later.

Reply:
I'm sorry. I've been multi-threading lately, and I got this question wrong.
Here need to use common mpu_regions.c files here to re-configure mpu section for cm7 cores. Due to cm7 speculative prefetches feature, we need to forbid cm7 core to access external memory. So I use common mpu_regions.c file under imxrt folder to re-config mpu regions.